cachematrix.R

```diff
@@ -1,15 +1,62 @@
-## Put comments here that give an overall description of what your
-## functions do
-
-## Write a short comment describing this function
-
-makeCacheMatrix <- function(x = matrix()) {
```

Comment:
Hello, your coding looks fine, it has no  mistakes. 
Keep up the great work
Regards,
Neha gupta